generate/templates/ping360.py.in

```diff
@@ -8,18 +8,14 @@
 # DO NOT EDIT
 # ~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!
 
-from brping import definitions
-from brping import PingDevice
-from brping import pingmessage
+from brping import definitions, PingDevice, PingMessage
 import time
 
 class Ping360(PingDevice):
+    messages = definitions.Ping360Message
+
     def initialize(self):
```

Comment:
Would be good to include a protocol version check here, that prints a warning about auto messages not being supported if the protocol is `< v1.1.0`.